src/game/trigger.cpp

```diff
@@ -391,6 +391,19 @@ void ActionStopTimer()
 	GameTimer.Running = false;
 }
 
+/**
+**  Remove a trigger
+**
+**  @param trig  Current trigger
+*/
+static void TriggerRemoveTrigger(int trig)
```

Comment:
Not your code, but using `lua_State* l` would avoid global.
Add comment that top stack should be the trigger table.

Reply:
fixed